contracts/arbitrators/KlerosLiquidProxy.sol

```diff
@@ -35,9 +44,30 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
         emit MetaEvidence(META_EVIDENCE_ID, _metaEvidence);
     }
 
-    function createDispute(uint256 claimIndex) external payable override {
-        (, , address claimant, , , , string memory evidence) = claimsManager
-            .claims(claimIndex);
+    function createDispute(
+        uint256 claimIndex,
+        bytes32 policyHash,
+        address claimant,
+        address beneficiary,
+        uint256 claimAmountInUsd,
+        string calldata evidence
+    ) external payable override {
+        (, , bytes27 claimHash) = claimsManager.claims(claimIndex);
+        require(
```

Comment:
Is this check necessary here seeing that the call further down to `claimsManager.createDispute(...)` will do this check as well?